### PR TITLE
Send receive fix

### LINKED ISSUE #632: receive statistics not properly re-initialized
Receive can be triggered multiple times in order to transfer several chunks into a table (represented by a query result window on the target table).

The statistics popup message window should ideally be cleared before attempting a new receive.
This is not implemented yet and (If it is not cleared manually) the statistics popup is not properly initialized:

image

This is the final result after sucessfully transferring 100'000 rows after 90'000 in the last round.

The query producing the transfer was:
```
select
    MSISDN, BARSCM, BARCU, SZENARIO
from
    csv$comma$header$skip200001."D:/Temp/smo-268-sbs-update.csv_00.csv"
where
    rownum <= 100000
```

### PR DESCRIPTION
**Pull request by shamis**
fixes #632 
- when a filter is changed on the sender the sender process is terminated
- random id for progress bar to show the status on the correct dialog
- getting the data_info from correct ets to show the stats correctly 

with help from @acautin

### CODE REVIEW

File: priv/dev/static/scripts/dderl.table.js [hunk 1]
```diff
@@ -963,42 +966,45 @@ const stmtClassToolTip = {
                     msg += ",<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;";
                     msg += activationResult.sender_columns[i];
                 }
-                msg += "<br>Progress: <span id='receivedRows'>0</span><div id='progressbar'></div><br><div id='receiverErrors'></div>";
+                var randomId = Math.floor((Math.random() * 1000) + 1);
+                this._sendReciveProgressId = randomId;
+                msg += "<br>Progress: <span id='receivedRows" + randomId + "'>0</span><div id='progressbar" + randomId + "'></div><br><div class='receiverErrors' id='receiverErrors" + randomId + "'></div>";
```
Comment (c-bik) [inline priv/dev/static/scripts/dderl.table.js]: please multiline this like an HTML block for better readblity

File: priv/dev/static/scripts/dderl.table.js [hunk 2]
```diff
@@ -963,42 +966,45 @@ const stmtClassToolTip = {
...
             }
         },
 
         _receiverStatus: function (maxRows) {
+            var randomId = this._sendReciveProgressId;
             this._ajax('receiver_status', {}, 'receiver_status', (receiverStatus) => {
                 if (receiverStatus.errors) {
                     for (let error of receiverStatus.errors) {
-                        $("#receiverErrors").append(`<span>Error : ${error}</span><br>`);
+                        $("#receiverErrors" + randomId).append(`<span>Error : ${error}</span><br>`);
```
Comment (c-bik) [inline priv/dev/static/scripts/dderl.table.js]: multiline for readablity

File: priv/dev/static/scripts/dderl.table.js [hunk 3]
```diff
@@ -963,42 +966,45 @@ const stmtClassToolTip = {
...
                     }
                 }
                 var receivedRows = receiverStatus.received_rows;
                 if (receiverStatus.is_complete) {
-                    $("#receivedRows").text(`completed, Rows successfully inserted : ${receivedRows}`);
-                    $("#progressbar").progressbar({
+                    $("#receivedRows" + randomId).text(`completed, Rows successfully inserted : ${receivedRows}`);
```
Comment (c-bik) [inline priv/dev/static/scripts/dderl.table.js]: multiline for readablity

**Comment by shamis** [comment]
have made the changes, please verify

**Pull request merged**